frontend/src/interfaces/featureToggle.ts

```diff
@@ -52,7 +52,7 @@ export interface IFeatureVariant {
     name: string;
     stickiness: string;
     weight: number;
-    weightType: string;
+    weightType: 'fix' | 'variable';
```

Comment:
isn't it gonna break openapi schemas?

Reply:
That's in frontend, before we where generating it from OpenAPI. Something to clean up and refactor